schemas/questionnaire_v1.json

```diff
@@ -59,6 +59,19 @@
         "StatisticsOfTradeAct"
       ]
     },
+    "view_submitted_responses": {
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "description": "Whether or not answers are available after submission.",
+          "type": "boolean"
+        },
+        "timeout": {
+          "description": "The amount of time in seconds submitted answers are available for.",
+          "type": "integer"
+        }
+      }
+    },
```

Comment:
Are both of these properties required?
we should also add `"additionalProperties": false,` so prevent additional properties being added to the object